lms/envs/devstack.py

```diff
@@ -43,6 +43,11 @@
 ANALYTICS_DASHBOARD_URL = None
 
 
+########################### PIPELINE #################################
+
+# Skip RequireJS optimizer in development
+STATICFILES_STORAGE = 'pipeline.storage.PipelineCachedStorage'
```

Comment:
Good catch. I'll remove that and we'll just have to remember to add it in when we add optimizer to LMS.